src/components/used-storage.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Progress } from 'antd';
+import PropTypes from 'prop-types';
+
+function convertBytesToHigherUnit(bytes) {
+  const kilobytes = bytes / 1000;
+  if (kilobytes < 1000) {
+    return `${kilobytes} KB`;
+  }
+  const megabytes = kilobytes / 1000;
+  if (megabytes < 1000) {
+    return `${megabytes} MB`;
+  }
+  const gigabytes = megabytes / 1000;
+  return `${gigabytes} GB`;
+}
+
+function UsedStorage({ usedBytes, maxBytes }) {
+  const format = percent => `${percent} %`;
+  const percent = usedBytes * 100 / maxBytes;
```

Comment:
We might want to take two edge cases into account here:
1. `maxBytes` is `0` then we would divide by zero
2. The scenario I described above (a user had private space and uploaded some files, then an admin takes unassigns the plan from the user), then the percentage will be higher than 100%, so we probably should use `Math.min(100, percent)` here

Reply:
Didn't consider covering this case since we don't have it yet, but it's not difficult and better be safe indeed.
Done, thanks!